dianna/utils/__init__.py

```diff
@@ -1,6 +1,2 @@
 # flake8: noqa: F401
-from .misc import get_function
-from .misc import get_kwargs_applicable_to_function
-from .misc import move_axis
-from .misc import onnx_model_node_loader
-from .misc import to_xarray
+from .misc import *
```

Comment:
In fact, the list became shorter at one point and then the same as it was before again... I will revert this.

Reply:
Ah now I remember. In methods, we removed __init__.py, because it is now a namespace package. We were thinking about how we could remove this __init__.py as well, but in this case it's less obvious. The onnx_runner.py submodule is separate so that non-users don't have to import it (and with it the heavy import on top). Because that file contains a class, you have to do a bit of an ugly hack if you want to import stuff inside the class and be able to use it from all methods. We did do it somewhere else, so we could reconsider. Then, everything in utils could just be in one `utils.py` file, and we can get rid of the overcomplicated directory structure with the `__init__.py` file.

Ok, all this doesn't actually explain the `*` after all, but this was our thought process :D Probably we just changed this because we were moving things around and didn't want to have to rewrite this all the time.